pkgs/os-specific/linux/libcap/default.nix

```diff
@@ -22,6 +22,8 @@ stdenv.mkDerivation rec {
   makeFlags = [
     "lib=lib"
     (stdenv.lib.optional (pam != null) "PAM_CAP=yes")
+    "BUILD_CC=${buildPackages.stdenv.cc.targetPrefix}gcc"
```

Comment:
Can we not force `gcc`?

Reply:
Done, also noticed this should be using depsBuildBuild and other minor cleanups.